src/Views/LoginPage/components/Form/Form.js

```diff
@@ -19,17 +23,19 @@ function Form(props) {
             type="email"
             autoComplete=""
             placeholder="email"
-            name="email"
+            name='email'
             value={props.initValues.email}
-            onChange={(e) => getValue(e.target.value, "email")}
+            onChange={(e) => setValue(e.target.value, "email")}
+            onFocus={(e) => setValidationError({ ...validationError, errorMessage:  emailValidation(e.target.value) })}
+            message={validationError.errorMessage}
```

Comment:
Please change it to 'errorMessage'

Reply:
do you expect to see message={errorMessage} ? instead